app/controllers/message_groupdates_controller.rb

```diff
@@ -0,0 +1,15 @@
+class MessageGroupdatesController < ApplicationController
+  def index
+    channel_id = params[:channel_id]
+    messages = channel_id.nil? ? Message.all : Channel.find(channel_id).messages
```

Comment:
これだと http://localhost:4000/message_groupdate?channel_id= にアクセスすると `channel_id` が空文字になり `channel_id.nil?` が偽になるため、以下のように [blank?](https://guides.rubyonrails.org/active_support_core_extensions.html#blank-questionmark-and-present-questionmark) を使うと良いかなと思いました。

```suggestion
    messages = channel_id.blank? ? Message.all : Channel.find(channel_id).messages
```

Reply:
ありがとうございます！